domains/measurements/Measurements.API/API/Helpers/Configuration.cs

```diff
@@ -2,11 +2,11 @@ namespace API.Helpers
 {
     public static class Configuration
     {
-        const string DataSyncEndpoint = "DATASYNCENDPOINT";
+        const string dataSyncEndpoint = "DATASYNCENDPOINT";
```

Comment:
Shouldn't this be PascalCase?

Reply:
@robertrosborg - The rules we have setup up says the public and protected should be PascalCase and everything else is camelCase (rules: `dotnet_naming*`). That string is (implicitly) private. We did discuss enforcing that implicit modifiers always be present (rule: `dotnet_style_require_accessibility_modifiers`), but opt (so far) for having it set to the default.